utilities/transactions/transaction_base.cc

```diff
@@ -253,8 +248,12 @@ Status TransactionBaseImpl::Get(const ReadOptions& read_options,
 Status TransactionBaseImpl::Get(const ReadOptions& read_options,
                                 ColumnFamilyHandle* column_family,
                                 const Slice& key, PinnableSlice* pinnable_val) {
-  return write_batch_.GetFromBatchAndDB(db_, read_options, column_family, key,
-                                        pinnable_val);
+  ReadOptions complete_read_options(read_options);
+  if (complete_read_options.io_activity == Env::IOActivity::kUnknown) {
+    complete_read_options.io_activity = Env::IOActivity::kGet;
+  }
+  return write_batch_.GetFromBatchAndDB(db_, complete_read_options,
+                                        column_family, key, pinnable_val);
```

Comment:
OK I can see why input validation needed to change based on this call. However it looks like it could have changed from:

```
  if (read_options.io_activity != Env::IOActivity::kUnknown) {
// error
```

to

```
  const IOActivity kMyIOActivity = kGet;
  if (read_options.io_activity != Env::IOActivity::kUnknown &&
      read_options.io_activity != kMyIOActivity) {
// error
```

Although this may be irrelevant depending on the answer to the `InternalReadOptions` question

Reply:
Fixed